src/main/java/com/locallampoon/fiveh/core/Player.java

```diff
@@ -59,20 +60,19 @@ void addItem(String item) {
         // Players will be able to add Items to their inventory until the maximum items are reached
         if (item.equals("duffel")) {
             changeInventorySize();
-            narrativePanel.appendTextArea("You now have a Duffel Bag, you can carry twice as many items!\n", FG_COLOR);
+            actionPanel.appendTextArea("You now have a Duffel Bag, you can carry twice as many items!\n", FG_COLOR);
             printInventoryItems();
         } else if (inventory.contains("")) {
             int index = inventory.indexOf("");
             inventory.remove(index);
             inventory.add(index, item);
-            if (narrativePanel != null) {
-                narrativePanel.appendTextArea("You added " + item + " to your inventory\n", FG_COLOR);
+
+            if (actionPanel!=null) {
```

Comment:
i'm seeing inconsistent white-spacing across the entire file. you can do a code format to fix this

Reply:
Will fix that in next PR